src/Store.js

```diff
@@ -2,42 +2,114 @@ import {
   observable,
 } from 'mobx';
 
-class EmptyTransporter {}
+import DefaultItem from './Item';
+import EmptyTransporter from './Transporters/EmptyTransporter';
+import { ROLE, SOURCE, STATE } from './constants';
 
 export default class Store {
 
-  static transporterOptions = {
-    http: {
-      url: 'http',
-    },
+  @observable items = [];
+  @observable loaded = false;
+  // @observable deletedItems = [];
+  constructor({
+    Item = DefaultItem, schema,
```

Comment:
I haven't found schema at first glance. Maybe we should put all attributes in a separate line when there are multiple lines of attributes?